ecs/README.md

```diff
@@ -4,9 +4,8 @@ This script generates the ECS mappings for the Wazuh indices.
 
 ### Requirements
 
-- ECS repository clone. The script is meant to be launched from the root level of that repository.
-- `Python` 3.6 or higher + `venv` module
-- `jq`
+- [Docker Desktop](https://docs.docker.com/desktop/setup/install/linux/)
```

Comment:
Is it Docker Desktop actually a requirement? I think `docker compose` is, which can be installed without Docker Desktop.

Prefer docker and docker compose plugin instead, as it's a slimmer installation and less problematic (we'be had problems with Docker Desktop in the past).

Reply:
Noted! Updated, now the dependency is just Docker Compose